languages/csharp/exercises/concept/namespaces/.docs/introduction.md

```diff
@@ -0,0 +1,34 @@
+Namespaces are a way to avoid name clashes and are generally present in all but the most trivial code base.
+
+The syntax is as follows:
+
+```csharp
+namespace MyNameSpace
+{
+    public class MyClass {}
+
+    public class OtherClass {}
+}
+```
+
+Types enclosed in namespaces are referred to outside the namespace by prefixing the type name with the dot syntax. Alternatively, and more usually, you can place a `using` directive at the top of the file (or within a namespace) and type can be used without the prefix. Within the same namespace there is no need to qualify type names.
+
+```csharp
+namespace MySpace
+{
+    public MyClass {}
+
+    new MyClass();
```

Comment:
This doesn't actually compile I think. I'm wondering if we could have a compiling example, although that would add the number of lines of code.

Reply:
Indeed.  It now compiles.  I ran out of prebuilt metasyntactic variables but soldiered on.